index.html

```diff
@@ -6,6 +6,6 @@
     <title>Welcome to Potionfy</title>
 </head>
 <body>
-
+	<h1>Welcome to Potionfy! Enjoy your stay</h1>
```

Comment:
Can you remove the 'Enjoy your stay.' string and mame the 'Welcome to Potionfly!' string red ?